course-mw/course-actors/src/main/java/org/sunbird/learner/actors/coursebatch/CourseBatchManagementActor.java

```diff
@@ -837,4 +839,65 @@ private boolean batchDatesUpdateNotificationActive() {
             PropertiesCache.getInstance()
                     .getProperty(JsonKey.SUNBIRD_BATCH_UPDATE_NOTIFICATIONS_ENABLED));
   }
+
+  private void createEventBatch(Request actorMessage) throws Throwable {
```

Comment:
is this code required ??

Reply:
not required, i have removed (earlier code)